### PR TITLE
Deleting a user should show name

### PR DESCRIPTION
**Pull request by Chris-Codes-It**
**Changes**

Amended Delete User dialog title to include username of user to be deleted

**Issues**

https://features.jellyfin.org/posts/2392/deleting-a-user-should-show-name

### CODE REVIEW

**Comment by Chris-Codes-It** [comment]
I've made the changes to the required UI files, and the English strings files. I've left the other language files alone as I would not be able to accurately translate

**Comment by Chris-Codes-It** [comment]
image

**Comment by Chris-Codes-It** [comment]
image

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -138,3 +138,4 @@
  - [jomp16](https://github.com/jomp16)
  - [Leon de Klerk](https://github.com/leondeklerk)
  - [CrispyBaguette](https://github.com/CrispyBaguette)
+ - [Chris-Codes-It](https://github.com/Chris-Codes-It)
```
Comment (dmitrylyzo) [inline CONTRIBUTORS.md:141]: This is Emby contributors list. Move to the list above.

Comment (Chris-Codes-It) [reply CONTRIBUTORS.md]: Done

**Review by Gauvino — APPROVED** [review]

File: src/apps/dashboard/routes/users/index.tsx [hunk 2]
```diff
@@ -49,12 +49,18 @@ const UserProfiles: FunctionComponent = () => {
...
             const card = dom.parentWithClass(elem, 'card');
             const userId = card?.getAttribute('data-userid');
+            const username = card?.getAttribute('data-username');
 
             if (!userId) {
                 console.error('Unexpected null user id');
                 return;
             }
 
+            if (!username) {
```
Comment (ptalmeida) [inline src/apps/dashboard/routes/users/index.tsx:59]: Do we need to return if no username exists? if we only need the userId I think this check isn't necessary and could prevent the deletion of some buggy user with a missing username.

**Review by ptalmeida — COMMENTED** [review]
I'd recommend squashing all those commits into one

**Review by thornbill — APPROVED** [review]

File: CONTRIBUTORS.md [hunk 3]
```diff
@@ -56,6 +56,62 @@
 - [is343](https://github.com/is343)
 - [Meet Pandya](https://github.com/meet-k-pandya)
 - [Peter Spenler](https://github.com/peterspenler)
+ - [JoshuaBoniface](https://github.com/joshuaboniface)
```
Comment (dmitrylyzo) [inline CONTRIBUTORS.md:59]: Unrelated changes due to a wrong merge conflict resolution.

**Review by dmitrylyzo — CHANGES_REQUESTED** [review]

**Review by dmitrylyzo — APPROVED** [review]

File: CONTRIBUTORS.md (hunk 3 above)
Comment (thornbill) [reply CONTRIBUTORS.md]: Yeah... the automerging seemed to do some weird things when rebasing... I've fixed it now and squashed the commits
